### PR TITLE
wording fixes, simplify some strings and fix capitalization

### PR DESCRIPTION
**Pull request by jancborchardt**
Fix some wording, mostly:

- fix capitalization to only capitalize first word of any given action. This is how it should always be.
- change »Switch to list/grid view« to simpler »Grid/List view«.
- fixed various technical instances of words like »URL« etc.

Please review @tobiasKaminsky @AndyScherzinger 

@davivel @masensio @rperezb please make sure we also fix this in the iOS app.

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
Changed look good to me :+1:

**Comment by tobiasKaminsky** [comment]
Isn't this also done by transifex? Or is the default one managed by github?

**Comment by tobiasKaminsky** [comment]
Beside that, changes look good to me :+1:

**Comment by jancborchardt** [comment]
@tobiasKaminsky the original strings (usually English) are always in the repository. Transifex needs a source for that, and as visible in the files the format is also slightly different for the English ones. :)

File: res/values/strings.xml [hunk 1]
```diff
@@ -64,7 +64,7 @@
     <string name="uploader_btn_new_folder_text">New folder</string>
     <string name="uploader_top_message">Choose upload folder:</string>
     <string name="uploader_wrn_no_account_title">No account found</string>
-    <string name="uploader_wrn_no_account_text">There are no %1$s accounts on your device. Please setup an account first.</string>
```
Comment (LukeOwncloud) [inline res/values/strings.xml]: AFAIK, "setup" is not a verb. It is "to set" (up).

**Comment by rperezb** [comment]
@tobiasKaminsky yes, original or base language is changed in this repo, other languages...Transifex

**Comment by davivel** [comment]
:+1:

**Pull request merged**